meetings/2024-09-17.md

```diff
@@ -0,0 +1,10 @@
+# TC Meeting 2024-09-17
+
+Present: Frederik, Max, Loic, Florian, Tristram, Cornelius, Stephanie, Adrian; Excused: 
+
+## Agenda
+
+* Report of TC at board meeting on September 26
+* Onboarding (https://github.com/OpenRailAssociation/technical-committee/labels/Onboarding)
+* OSRD stage 2 proposal: https://github.com/OpenRailAssociation/technical-committee/pull/141
+  
```

Comment:
I'd like to add a topic: Conferences which are relevant for users/creators of current/future OpenRail projects